src/tests/ftest/security/pool_connect_init.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2017-2019 Intel Corporation.
```

Comment:
Probably just 2019 since it's a new file

Reply:
Thanks for the review and comments. 
Will update.